### PR TITLE
github action for pages deploy

### PR DESCRIPTION
**Pull request by briekman**
adding workflow to build the pages and publish

couple of settings required  shown below

give the token access to publish to the repo and once its made a branch select the gh-pages branch as the deploy branch in the pages setting

### CODE REVIEW

File: .github/workflows/pages.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Your GitHub workflow file under .github/workflows/
+#
+
+name: Pages
+on: [push]
+jobs:
+  publish-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Chekout
```
Comment (JamJar00) [inline .github/workflows/pages.yaml]: Typo! 
```suggestion
      - name: Checkout
```

**Review by JamJar00 — CHANGES_REQUESTED** [review]

**Review by JamJar00 — APPROVED** [review]

**Comment by JamJar00** [comment]
New policy is for two mods to approve so will prod someone to give you another approval!

**Review by MrBabadook — APPROVED** [review]
Seems good - will have to update them in the future to fix Configuration not pointing at a valid URL but thanks for getting this built :)

**Comment by briekman** [comment]
removed the on push and added workflow_dispatch so we can manually trigger instead

**Pull request closed without merging**